src/Header.js

```diff
@@ -50,8 +51,12 @@ const Header = () => {
             )}
           </NavItem>
           <NavItem>
-            <TopNavLink href="./">
-              <FaShoppingCart />
+            <TopNavLink href="/cart">
+              <FaShoppingCart />({" "}
+              {parseInt(globalCart.cartProducts.reduce((count, curItem) => {
+                return parseInt(count) + parseInt(curItem.cartQty);
+              }, 0)) }{" "}
```

Comment:
```suggestion
            <FaShoppingCart />({" "}
              {globalCart.cartProducts.reduce((count, curItem) => {
                return count + parseInt(curItem.cartQty);
              }, 0) }{" "}
              )
```
Actually we don't need to parse all this values, just the quantity that is inside the item object, since you indicated to the reduce function that the initial value is a number